server/monitor/src/main/java/org/apache/accumulo/monitor/rest/trace/TracesResource.java

```diff
@@ -77,12 +81,12 @@
    * Generates a trace summary
    *
    * @param minutes
-   *          Range of minutes to filter traces
+   *          Range of minutes to filter traces Min of 0 minutes, Max of 30 days
    * @return Trace summary in specified range
    */
   @Path("summary/{minutes}")
   @GET
-  public RecentTracesList getTraces(@DefaultValue("10") @PathParam("minutes") int minutes) throws Exception {
+  public RecentTracesList getTraces(@DefaultValue("10") @PathParam("minutes") @Min(0) @Max(2592000) int minutes) throws Exception {
```

Comment:
The annotations are part of the javax.validation api and Jersey 2.0+ has built in support for them.  According to the documentation the auto-discovery service will auto-enable them on the server without the need of manual configuration.  The reference implementation is the hibernate validator which will need to be explicitly added to the pom based on their documentation:

https://jersey.github.io/documentation/latest/bean-validation.html

I'm still reading up on it and it sounds like it could negate the need for the ParameterValidator class in this PR (which I'll end up removing).